plasmapy/physics/tests/test_parameters.py

```diff
@@ -477,39 +476,33 @@ def test_gyrofrequency():
 def test_gyroradius():
     r"""Test the gyroradius function in parameters.py."""
 
-    assert gyroradius(B, T_e).unit == u.m
+    assert gyroradius(B, T_i=T_e).unit == u.m
```

Comment:
I'm thinking we need to do a code review for tests, especially for `parameters.py`, as we just have poorly bundled asserts under single functions. Basically we don't want failure of the first assert to prevent testing the remaining asserts for a function. I personally like the class-method approach, but we can discuss what is optimal.

Reply:
We do. We absolutely do.

Reply:
I'll add it to #234